extension/src/background/constants/apiUrls.ts

```diff
@@ -1,5 +1,2 @@
 export const STELLAR_EXPERT_BLOCKED_ACCOUNTS_URL =
```

Comment:
we're still going to use stellar.expert to check for `memo required` accounts to warn users if an xdr will require a memo.

This var is a misnomer now - I'm going to rename it to `STELLAR_EXPERT_MEMO_REQUIRED_ACCOUNTS_URL` in the next PR. Holding off because this PR is already pretty large